cdk/bin/cdk.ts

```diff
@@ -0,0 +1,25 @@
+import 'source-map-support/register';
+import {GuRoot} from '@guardian/cdk/lib/constructs/root';
+import {Gatehouse} from '../lib/gatehouse';
+
+const app = new GuRoot();
+
+const stack = 'identity';
+
+const env = {
+    region: 'eu-west-1',
+};
+
+new Gatehouse(app, 'gatehouse-CODE', {
+    stack,
+    env,
+    stage: 'CODE',
+    domainName: 'gatehouse.code.dev-guardianapis.com',
+});
+
+new Gatehouse(app, 'gatehouse-PROD', {
+    stack,
+    env,
+    stage: 'PROD',
+    domainName: 'gatehouse.guardianapis.com',
```

Comment:
I guess these will change or be supplemented by the `identity.` domain names later?

Reply:
Updated domain names so that it's clearer they are internal backends and following similar format to gateway in 8e3e4755e5b1731e04e723ab5807158c83801d1a